src/components/Main.js

```diff
@@ -14,10 +14,9 @@ import '../css/main.css';
 
 // Megahack to include font-awesome.
 injectCSS('https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css');
-injectCSS('https://fonts.googleapis.com/css?family=Roboto:400,300,500');
 
 export default class Main extends React.Component {
-  constructor (props) {
```

Comment:
did you make these changes, or did your editor? based on Standard/Semistandard rules, this is actually correct to keep a space before